recipes/knoten/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "knoten" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/USGS-Astrogeology/knoten/archive/{{ version }}.tar.gz
+  sha256: d671f1c8c32107cf1ef1d2dadf236ba98f9f19dcfb744e302cb20860748164b0
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [ win or py2k ]
+  number: 0
+
+requirements:
+  build:
+  host:
+    - python >=3.7,<3.8
```

Comment:
This seems quite restrictive. Are you sure it doesn't work with 3.8 and above?

Reply:
yeah, I've only spent a short time looking into it, but on of it's dependencies is generated via swig wrappers and something about how it's configured only allows for 3.7. Maybe lower, but definitely not higher. I wasn't able to get an environment with anything higher anyways.

Reply:
If this project itself doesn't have any code not compatible with Python >=3.7, you shouldn't explicitely set an upper bound.
If a dependency requires Python 3.7, that dependency will constrain the Python version. And if a new version is released for another Python version, your package will just work without you having to update it.

You can remove the `<3.8`.